### PR TITLE
RefreshManager fix

### PR DESCRIPTION
**Pull request by barijaona**
Fix update of database when permanent redirection of a feed is applied
Other small fixes

### CODE REVIEW

File: Vienna/Sources/Fetching/RefreshManager.m [hunk 1]
```diff
@@ -99,7 +99,7 @@ -(instancetype)init
         config.HTTPAdditionalHeaders = @{@"User-Agent": userAgent};
         config.HTTPMaximumConnectionsPerHost = 6;
         config.HTTPShouldUsePipelining = YES;
-        _urlSession = [NSURLSession sessionWithConfiguration:config delegate:self delegateQueue:[NSOperationQueue mainQueue]];
+        _urlSession = [NSURLSession sessionWithConfiguration:config delegate:self delegateQueue:nil];
```
Comment (Eitot) [inline Vienna/Sources/Fetching/RefreshManager.m:102]: Is this not going to cause any problems with the delegate callbacks? The delegate queue would probably a private background queue.

Comment (barijaona) [reply Vienna/Sources/Fetching/RefreshManager.m:102]: > The queue should be a serial queue, in order to ensure the correct ordering of callbacks. If `nil`, the session creates a serial operation queue for performing all delegate method calls and completion handler calls.

So this change should not affect the sequence of delegate callbacks.

**Comment by barijaona** [comment]
As you aren't completely comfortable with a default background queue, I removed this change (forced update)

**Review by Eitot — APPROVED** [review]
Sorry, you didn’t have to change it!

**Comment by barijaona** [comment]
So , do you prefer with or without the change of operation queue ? I was comfortable with the change, but have no problem with the status quo if anyone is more comfortable with it.

**Comment by Eitot** [comment]
It was no objection on my end, just something that crossed my mind. If you see no concerns, I have no problems if you make the change.

**Review by Eitot — APPROVED** [review]

**Pull request merged**